app/routes/oauth-stripe.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Route
+} = Ember;
+
+export default Route.extend({
+  // model(params) {
```

Comment:
Yeah, I'm indifferent. If someone really wants to look up what params are coming through, they can look at the `queryParams` hook. It isn't like we need to protect against params that aren't `code` or `state`.  I'm fine either way!